test/WebJobs.Script.Tests/IO/AutoRecoveringFileSystemWatcherTests.cs

```diff
@@ -82,9 +82,12 @@ public void Delete_SendsExpectedNotification()
                 // 1 trace per attempt + 1 trace per failed attempt
                 int expectedTracesBeforeRecovery = (expectedNumberOfAttempts * 2) - 1;
                 // Before + recovery trace
-                int expectedTracesAfterRecovery = expectedTracesBeforeRecovery + 1;
+                int expectedTracesAfterRecovery = expectedTracesBeforeRecovery + 2;
```

Comment:
We now log even on the first interval where there is no wait, whereas previously we only logged if we were delaying.